client/src/components/EmailChange/EmailChange.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import '../styles/style.scss';
+import { Button, Form, FormProps, Input } from 'reactstrap';
+import '../styles/Profile.scss';
+
+export default class EmailChange extends React.Component<any> {
```

Comment:
No need to use class for simple component

Use function

Reply:
Fixed